schema_enforcer/cli.py

```diff
@@ -303,8 +303,9 @@ def ansible(
                 data = hostvars
 
             # Validate host vars against schema
-            for result in schema_obj.validate(data=data, strict=strict):
+            schema_obj.validate(data=data, strict=strict)
```

Comment:
Please can you make the same changes to the `validate` command , otherwise validators won't work for this command

Reply:
The validate CLI command calls validate on the InstanceFile object. The InstanceFile validate method was refactored for the new approach:

https://github.com/networktocode/schema-enforcer/blob/7d557ef0d14dac8a26993f3ce0c7f871068db5c3/schema_enforcer/instances/file.py#L128-L134